### PR TITLE
Update minisatip version

### PR DESCRIPTION
**Pull request by catalinii**

### CODE REVIEW

File: packages/addons/service/minisatip/source/bin/minisatip.start [hunk 1]
```diff
@@ -32,7 +32,7 @@ fi
 
 # debug log
 if [ "$MINISATIP_DEBUG" = "true" ]; then
-  MINISATIP_ARG="-l -l -l"
+  MINISATIP_ARG="-l http"
```
Comment (CvH) [inline packages/addons/service/minisatip/source/bin/minisatip.start]: wasn't aware that this is possible now, I add some followup pr that adds the ability to select the debug "modules" at the gui like we done it at Tvheadend

File: packages/addons/service/minisatip/package.mk [hunk 2]
```diff
@@ -2,9 +2,8 @@
 # Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="minisatip"
-PKG_VERSION="a3af6dd93c4fd9dbbb62073f3642a289826d84bb"
-PKG_SHA256="32d17edab044f9fdba9e39bb4a8332aa087951695c1629d81687ff957c8d601d"
-PKG_VERSION_NUMBER="0.7.18"
+PKG_SHA256="04d208369331ef0ee165766438a80ed826bb6a6552bbc56a38a6f2dd04603da3"
+PKG_VERSION="1.0.3"
 PKG_REV="101"
```
Comment (CvH) [inline packages/addons/service/minisatip/package.mk]:
```suggestion
PKG_REV="102"
```
to bump the addon version, also an entry at the changelog.txt is needed

File: packages/addons/service/minisatip/package.mk [hunk 3]
```diff
@@ -2,9 +2,8 @@
 # Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="minisatip"
-PKG_VERSION="a3af6dd93c4fd9dbbb62073f3642a289826d84bb"
-PKG_SHA256="32d17edab044f9fdba9e39bb4a8332aa087951695c1629d81687ff957c8d601d"
-PKG_VERSION_NUMBER="0.7.18"
+PKG_SHA256="04d208369331ef0ee165766438a80ed826bb6a6552bbc56a38a6f2dd04603da3"
+PKG_VERSION="1.0.3"
```
Comment (chewitt) [inline packages/addons/service/minisatip/package.mk:6]: please reorder so PKG_VERSION comes before PKG_SHA256 .. to be consistent with all other packages.

File: packages/addons/service/minisatip/source/bin/minisatip.start (hunk 1 above)
Comment (catalinii) [reply packages/addons/service/minisatip/source/bin/minisatip.start:35]: Sounds good.

Comment (catalinii) [reply packages/addons/service/minisatip/source/bin/minisatip.start:35]: Sounds good.

File: packages/addons/service/minisatip/package.mk (hunk 2 above)
Comment (catalinii) [reply packages/addons/service/minisatip/package.mk]: Addressed the comment.

File: packages/addons/service/minisatip/package.mk (hunk 3 above)
Comment (catalinii) [reply packages/addons/service/minisatip/package.mk:5]: Done.

**Comment by CvH** [comment]
@catalinii entry at the changelog is still missing :) https://github.com/LibreELEC/LibreELEC.tv/blob/master/packages/addons/service/minisatip/changelog.txt

**Comment by catalinii** [comment]
@CvH, sorry about that. I updated the file as well.
Thanks

**Pull request merged**